wizeline/wize-q-remix
# feat: dislike-question

#### What does this PR do?
- Adds the new functionality to dislike a question. 

#### How should this be manually tested?
_Note: You have to pull this branch to your local environment and run the project._ 
1. Login
2. Dislike a question. 
3. Apply Order by _popular_ and verify that it shows the question in the correct order according to the selection. 
4. Go to the details of a question and click on dislike option. 
5. Go back to the home page and apply the Order by again. 

#### What are the relevant tickets?
(Link to issues, related PR, JIRA issues, etc.)
Closes #102 

#### Screenshots

https://user-images.githubusercontent.com/97203617/208564233-fd025248-e10f-4d1d-99f1-f173f3b86499.mov
